src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -853,6 +860,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		return this.groupView.getIndexOfEditor(editor);
 	}
 
+	private lastSelectedEditor: EditorInput | undefined;
```

Comment:
What is this variable about and isn't it becoming stale easily if selection changes?